toolkit/tools/internal/packagerepo/repocloner/repocloner.go

```diff
@@ -22,7 +22,7 @@ type RepoPackage struct {
 // It is capable of generate a local repository consisting of a set of request packages
 // and their dependencies.
 type RepoCloner interface {
-	Initialize(destinationDir, tmpDir, workerTar, existingRpmsDir, toolchainRpmsDir string, usePreviewRepo bool, repoDefinitions []string) error
+	Initialize(destinationDir, tmpDir, workerTar, existingRpmsDir, toolchainRpmsDir string, usePreviewRepo bool, repoDefinitions []string, disableMarinerRepos bool) error
```

Comment:
Small request: could you re-order the arguments, so each type is grouped together and we can thus not repeat `bool`?